src/main/java/practice/CandidateValidator.java

```diff
@@ -1,5 +1,25 @@
-package practice;
-
-public class CandidateValidator {
-    //write your code here
-}
+package practice;
+
+import java.util.function.Predicate;
+import model.Candidate;
+
+public class CandidateValidator implements Predicate<Candidate> {
+
+    private static final int MIN_AGE_CANDIDATE = 35;
+    private static final int MIN_AGE_LIVED = 10;
+
+    private static final String CORRECT_NATIONALITY = "Ukrainian";
+
+    @Override
+    public boolean test(Candidate candidate) {
+
+        String[] periodsInUkr = candidate.getPeriodsInUkr().split("-");

```

Comment:
All magic numbers, strings, and symbols in your code should be constants.
[this article](https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names)
```suggestion
        String[] periodsInUkr = candidate.getPeriodsInUkr().split(SEPARATOR);
```